notebooks/photutils/03_aperture_photometry/03_aperture_photometry.ipynb

```diff
@@ -0,0 +1,991 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There's only so much you can learn from looking at a table of numbers, so let's explore alternate ways to examine these data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(8, 5))\n",
+    "\n",
+    "values = [phot.value for phot in phot_table['aperture_sum']]\n",
+    "logbins=bins = 10.**(np.linspace(-1, 2, 100))\n",
+    "plt.hist(values, bins=logbins)\n",
+    "\n",
+    "plt.yscale('log')\n",
+    "plt.xscale('log')\n",
+    "plt.title('Histogram of Source Photometry')\n",
+    "plt.xlabel(r'Flux Count Rate ({})'.format(xdf_image.unit.to_string('latex')))\n",
+    "plt.ylabel('Number of Sources')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(8, 5))\n",
+    "\n",
+    "plt.scatter(table['area'], values, alpha=0.5)\n",
+    "\n",
+    "plt.yscale('log')\n",
+    "plt.xscale('log')\n",
+    "plt.title('Count Rate v. Aperture Area')\n",
+    "plt.xlabel('Aperture Area [pixels$^2$]')\n",
+    "plt.xlabel(r'Flux Count Rate ({})'.format(xdf_image.unit.to_string('latex')))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "\n",
+    "<h3>Exercise:</h3><br>\n",
+    "\n",
+    "Re-calculate the photometry for these elliptical apertures - or just a subset of them - using the <code>subpixel</code> aperture placement method instead of the default <code>exact</code> method. How does this affect the count sum calculated for those apertures?\n",
+    "\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "# Aperture Corrections"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We've done photometry with some lovely apertures, but unfortunately even using elliptical apertures with unique sizes and orientations does not account for an important source of extraneous flux: the sky background.\n",
+    "\n",
+    "## Local Background Subtraction\n",
+    "\n",
+    "In the [background estimation notebook](../01_background_estimation/01_background_estimation.ipynb), we explored how to perform global background subtraction of image data with `photutils`. However, you can also use `photutils` to perform local background estimations for aperture corrections.\n",
+    "\n",
+    "To estimate the local background for each aperture, measure the counts within annulus apertures around (but not including!) each source. In our example, we defined elliptical apertures with `r = 3` to measure the counts within each source. To calculate the background for each source, let's measure the counts elliptical annuli between `r = 3.5` and `r = 5`."
```

Comment:
The XDF field is well out of my comfort zone (I work essentially exclusively with ground-based images of stars) but typically the gap between the aperture and the annulus is larger.

I'm a little confused about the units though, so if this is half the radius of the object then please disregard.

Reply:
Unfortunately I actually know very little about photometry, and have never actually done photometric analysis outside of a university lab setting, so I don't really have an opinion here. 

How much larger would you recommend making the gap? The units are in pixels.

Reply:
I'd make the inner radius of the annulus about twi the radius of your aperture (so 6 pixels in this case).